src/service/defaultStructure.service.ts

```diff
@@ -98,21 +98,25 @@ export class DefaultStructureService {
    * @param {string} [functionName] - function name
    * @memberof DefaultStructureService
    */
-  public createFunctionsFolder(functionName?: string, takeRoot = false): void {
+  public createFunctionsFolder(functionName: string, takeRoot = false): void {
     const path = takeRoot ? this.fileService.getRoot() : this.cwd;
-    this.functionName = functionName || 'exampleFunction';
-    this.fileService.copy(
-      join(
-        this.dirname,
-        '..',
-        '..',
-        'bin',
-        'example',
-        'functions',
-        'exampleFunction',
-      ),
-      join(path, 'functions', this.functionName),
-    );
+    this.functionName = functionName;
```

Comment:
This `this.functionName = functionName;` should be places within the L49 in `create(functionName?, update?: boolean): void`. Because here it is implicit knowledge especially since later you check for the presents of `this.functionName`. Which was never set in the `create`-Method flow.